src/pretix/base/signals.py

```diff
@@ -787,3 +787,19 @@ class and point to a valid JPEG or PNG file. If no image is available, ``evaluat
 
 As with all event-plugin signals, the ``sender`` keyword argument will contain the event.
 """
+
+customer_created = GlobalSignal()
+"""
+Arguments: ``customer``
+
+This signal is sent out every time a customer account is created. The ``customer``
+object is given as the first argument.
```

Comment:
We should document what the ``sender`` will be, and I'd argue we should send the ``Organizer``. That way, we have a path towards future compatibility if we ever add better support for organizer-level plugins